tests/ci/fast_test_check.py

```diff
@@ -32,10 +32,10 @@ def get_fasttest_cmd(
         "--network=host "  # required to get access to IAM credentials
         f"-e FASTTEST_WORKSPACE=/fasttest-workspace -e FASTTEST_OUTPUT=/test_output "
         f"-e FASTTEST_SOURCE=/repo "
-        f"-e FASTTEST_CMAKE_FLAGS='-DCOMPILER_CACHE=sccache' "
+        f"-e FASTTEST_CMAKE_FLAGS='-DCOMPILER_CACHE=chcache' "
```

Comment:
I suggest to have sccache as failover cache, say if CHCACHE_PASSWORD (or better add new GH variable) is not set - then we use sccache. this way we can quickly switch CI between cache types